nucliadb/nucliadb/ingest/orm/processor/__init__.py

```diff
@@ -277,6 +280,11 @@ async def process(
                     else None
                 )
         elif message.type == BrokerMessage.MessageType.MULTI:
+            # XXX Not supported right now
+            # MULTI, COMMIT and ROLLBACK are all not supported in transactional mode right now
+            # This concept is probably not tenable with current architecture because
+            # of how nats works and how we would need to manage rollbacks.
+            # XXX Should this be removed?
```

Comment:
for me, we can remove all of this logic if it's not used.
I am OK if we do it in a separate PR though